api/dataservices/edgejob/edgejob.go

```diff
@@ -2,8 +2,7 @@ package edgejob
 
 import (
 	"fmt"
-
-	portainer "github.com/portainer/portainer/api"
```

Comment:
This is what this commit is all about - removing the link from this small submodule to the definition of everything - and really should be for all structures, so that it becomes obvious what code depends on what other code.